rachelnavarro/step/walkthroughs/week-4-libraries/maps/examples/google-tour/src/main/webapp/index.html

```diff
@@ -2,7 +2,7 @@
 <html>
 <head>
   <title>Google Tour</title>
-  <script src="https://maps.googleapis.com/maps/api/js?key=YOUR_API_KEY"></script>
+  <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyAjbN6eIDwFuoskbJSlQT2kM0bVnqyKGFw"></script>
```

Comment:
Do NOT check this in https://cloud.google.com/docs/authentication/api-keys#securing_an_api_key

I would recommend, if it's easy, to regenerate your key. I don't care if it's the walk through tells you to include the key it's extremely wrong.